### ISSUE TITLE
Fertilizer & Specialized parts required issue

### ISSUE THREAD

**Issue by OakTree42** [BobPalmer/MKS#728]
Right now basically to build things off planet or keep a colony going you need fertilizer and specialized parts. Is this intentional? Seems to mean that once you are out of specialized parts you're done building a colony out anymore with EPL, and once out of fertilizer you're stuck with a limited LS option that will seem to always work at a pretty hefty net loss. 

I understand this may be intentional but it does seem rather limiting compared to the previous system. Especially with the specialized parts and trying to build say on Eve or Duna or Dres etc. 

So to sum up the request, is there or will there be a way to create fertilizer or specialized parts off kerbin at some point (Yes, I know you can get some specialized parts by salvaging but that is a net loss game in rebuilding and limits colony options).

One suggestion that comes to mind is adding a converter to the UKS Training Akademy that would allow conversion of material kits to specialized parts at say a 10 to 1 conversion rate, plus power etc. You'd still need to get the akademy there but it would at least be some option.

**Comment by zajc3w** [comment]
Specialized parts are not required for mk3 fabricator module. it will allow to go directly from metal,chemicals and polymers to rocket parts and machinery.
I'm unsure about fertilizer, on current release, but in 31.1 there was an non fertilizer route to get supplies even with mkV modules.

**Comment by OakTree42** [comment]
I think you missed my point. EPL requires 25% by weight specialized parts to material kits in the latest USI update. The Fabrication module requires 2000 material kits (2000kg) and 132.3 specialized parts (500 kg)

And no module can create specialized parts at the moment. So yes, I can make material kits, but I can't /use/ them without specialized parts under the current EPL/USI system. In addition yes, you can gain specialized parts by scavenging instead of salvaging, but only equal to the number used in the construction in the first places so while that lets you re-allocate some specialized parts it does not let you actually 'grow' a colony without shipping in specialized parts.  This greatly limits the EPL system at the moment. And this is what I was pointing to.

As for the fertilizer, I eagerly look forward to see how that plays out, especially in relation to TAC-LS systems!

**Comment by BobPalmer** [comment]
Closing as the new parts are out :)
